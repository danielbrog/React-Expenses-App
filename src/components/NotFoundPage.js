import React from 'react'
import { Link} from 'react-router-dom'

const NotFoundPage = () => (
    <div>
        404 - <Link to="/home">Go back home.</Link>
    </div>
)

export default NotFoundPage