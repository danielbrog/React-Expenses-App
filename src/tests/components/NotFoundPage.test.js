import React from 'react'
import { shallow } from 'enzyme'
import NotFoundPage from '../../components/NotFoundPage'

test('Testing not found page page', () => {
    const wrapper = shallow(<NotFoundPage />)
    expect(wrapper).toMatchSnapshot()
})