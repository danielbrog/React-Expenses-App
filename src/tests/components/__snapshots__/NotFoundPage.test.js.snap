// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing not found page page 1`] = `
<div>
  404 - 
  <Link
    to="/"
  >
    Go back home.
  </Link>
</div>
`;
